### ISSUE TITLE
Quickly accessing many farm sites after daemon boot up crashes server

### ISSUE THREAD

**Issue by almereyda** [fedwiki/wiki#112]
Opening a page with [a large roster that points only to sites on the same farm](http://jon.2017.federated.wiki/welcome-visitors.html) crashes the server after loading about half of the flags. Triaging of this error has been [logged in the chat](https://matrix.to/#/!ORfrUEFeWFcHAMLFLr:matrix.org/$152332014121682IZyYN:matrix.allmende.io). The error presented is very ambiguous:

```

 Error: getaddrinfo ENOTFOUND raw.githubusercontent.com raw.githubusercontent.com:443
    at errnoException (dns.js:28:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:76:26)


server unexpectly exitted, 11 (1)
```

The service is deployed in a Docker container which from within can perfectly fine ping the DNS address in question. The daemon process terminates upon the client trying to fetch the favicons for display in the roster, while the server tries to start a new application for each separate site.

![bildschirmfoto vom 2018-04-10 02-42-50](https://user-images.githubusercontent.com/1645308/38548742-cc86d6c0-3cb2-11e8-819e-b917e7b7f592.png)

Funnily enough, the server still manages/tries/logs to boot up two sites after the message of unexpectedly exiting. If the page with the roster is left open, and the sites are opened lazily after a restart of the daemon, i.e. by slowly clicking their flags in the open roster one by one, reloading the page does not crash the server anymore. Yet the error reappears when after a clean restart trying to quickly open many sites after another from that roster.

Citing conclusions from a maintainer in the chat:

> this could be hard for the server to start while fetching many favicons for other sites

> We might not be able to make a server do the right thing while starting but it shouldn't die.

**Comment by paul90** [comment]
From the matrix conversation, the site being accessed looks to be http://jon.2017.federated.wiki/view/welcome-visitors.

The roster is not particularly large, having only 12 sites.

Are you using `https` to access the site? I notice that it has an invalid certificate, but not sure that explains the error message you are seeing.

**Comment by paul90** [comment]
Closing, this should have been fixed by fedwiki/wiki-plugin-plugmatic#7
